### PR TITLE
fix(widget-data-table): check ellipsis on table resizing

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue [hunk 1]
```diff
@@ -66,7 +68,7 @@
                                 @click="handleClickRow({rowIndex, item})"
                             >
                                 <td v-for="(field, colIndex) in props.fields"
-                                    :key="`td-${props.widgetKey}-${rowIndex}-${colIndex}`"
+                                    :key="`td-${props.widgetKey}-${rowIndex}-${colIndex}-${tableWidth}`"
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue:71]: praise: wow

**Review by yuda110 — APPROVED** [review]
LGTM. Very impressive way

File: apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue:71]: Question: I can't fully understand. Why did you use `tableWidth` in key?

**Review by piggggggggy — APPROVED** [review]

**Review by skdud4659 — APPROVED** [review]
LGTM!

File: apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue:71]: > Question: I can't fully understand. Why did you use tableWidth in key?

To rerender that element when `tableWidth` is updated so the tooltip can be shown when it is ellipsised.

Comment (sulmoJ) [reply apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue:71]: I think she was expecting to re-render when the key changed..?!

**Review by sulmoJ — APPROVED** [review]
GOOOOD

**Pull request merged**